dependencies/FindFastJet.cmake

```diff
@@ -46,43 +46,47 @@ if(${PKGNAME}_CONFIG)
                   ERROR_VARIABLE error
                   OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-  set(${PKGNAME}_LIBS)
-  set(${PKGNAME}_LIB_DIRS)
-  mark_as_advanced(${PKGNAME}_LIBS ${PKGNAME}_LIB_DIRS)
-  if(${PKGNAME}_CONFIGLIBS)
-    string(REGEX MATCHALL "(^| )-l[^ ]+" libs ${${PKGNAME}_CONFIGLIBS})
-    foreach(lib ${libs})
-        string(STRIP ${lib} lib)
-        string(SUBSTRING ${lib} 2 -1 lib)
-        list(APPEND ${PKGNAME}_LIBS ${lib})
-    endforeach()
-
-    string(REGEX MATCHALL "(^| )-L[^ ]+" libdirs ${${PKGNAME}_CONFIGLIBS})
-    foreach(libdir ${libdirs})
-        string(STRIP ${libdir} libdir)
-        string(SUBSTRING ${libdir} 2 -1 libdir)
-        list(APPEND ${PKGNAME}_LIB_DIRS ${libdir})
-    endforeach()
-
-    # append directories of exposed libraries
-    if ("CGAL" IN_LIST ${PKGNAME}_LIBS)
-      find_library(lib_cgal NAMES "CGAL" PATHS $ENV{CGAL_ROOT}/lib NO_DEFAULT_PATH)
-      if (NOT lib_cgal)
-        message(FATAL_ERROR "CGAL not found in $ENV{CGAL_ROOT}/lib")
+  if (${PKGNAME}_FOUND})
```

Comment:
@qgp let me check if github produces the whitespace-supressed diff on this comment.

Reply:
no, did not work